src/OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -159,14 +156,11 @@ void LoadCaptureInfo(const CaptureInfo& capture_info, CaptureListener* capture_l
     return;
   }
 
-  absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> selected_functions;
-  for (const auto& function : capture_info.selected_functions()) {
-    const auto& module_it = module_map.find(function.loaded_module_path());
-    CHECK(module_it != module_map.end());
-    ModuleData module(module_it->second);
-    uint64_t address = function_utils::GetAbsoluteAddress(function, process, module);
-    selected_functions[address] = function;
+  absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> instrumented_functions;
+  for (const auto& function : capture_info.instrumented_functions()) {
```

Comment:
Given that in the next lines, you are using both of them, why not capture both variables by names using structure bindings?

Reply:
done.

Reply:
I reverted this change because it does not sit well with msvc2017 & clang7